src/components/Integrations/Shopify/Shopify.js

```diff
@@ -99,37 +82,57 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
     </>
   );
 
+  useInterval({
+    runOnStart: true,
+    delay: 20000,
+    callback: async () => {
+      const result = await shopifyClient.getShopifyData();
+      if (!result.success) {
+        setShopifyState({
+          error: <FormattedHTMLMessage id="validation_messages.error_unexpected_HTML" />,
+        });
+      } else if (result.value.length) {
```

Comment:
dont need if `result.value` exist?

Reply:
If `result.success` the idea is that there should be a `value` in result. Maybe I am missing something?